server/integration-tests/run_and_compile.sh

```diff
@@ -2,15 +2,67 @@
 
 set -e
 
+source log.sh
+
 # Switch Java to v8 in runtime from current context
 . switch_java_version.sh
 
+GEM_DIR=calabash-android
```

Comment:
Could you please move new logic from here to separate .sh file? This file is more like common file that executes required steps. If we will have separate file, we can add it to `fast.sh`